src/components/Calculator.js

```diff
@@ -6,12 +6,13 @@ function Calculator(props) {
   const { calcState, handleClick } = props;
   return (
     <>
-      <div className="display">
-        {calcState.result}
-        {calcState.function}
-        {calcState.next}
-      </div>
       <div className="btnsContainer">
+        <div className="display">
+          {calcState.result}
+          {calcState.function}
+          {calcState.next}
+        </div>
```

Comment:
- [ ] Great job so far, now even though were not supposed to fpay much attention to styling, we should follow the given layout of components in the given [wireframes](https://github.com/microverseinc/curriculum-react-redux/blob/main/math-magicians/project_full_website.md)

Given Wireframe
![image](https://user-images.githubusercontent.com/81932478/235727067-e310dab2-c2f6-4f98-af36-3db15e336d4e.png)

Your design
![image](https://user-images.githubusercontent.com/81932478/235727264-54afc2cd-3eba-4393-a386-a5e0c4ce21eb.png)

### _Kindly ensure your calculator is aligned to right and the text `Let's do some math` is present and aligned to the right in alignment with the top of the calculator_

Reply:
Thank you for the helpful review. Surely I will come up with solution